ssz/sedes/base.py

```diff
@@ -61,24 +79,37 @@ def consume_bytes(data: bytes, start_index: int, num_bytes: int) -> Tuple[bytes,
             continuation_index = start_index + num_bytes
             return data[start_index:start_index + num_bytes], continuation_index
 
-    @abstractmethod
-    def intermediate_tree_hash(self, value: TSerializable) -> bytes:
-        pass
+    #
+    # Tree hashing
+    #
+    def hash_tree_root(self, value: TSerializable) -> bytes:
+        return EMPTY_CHUNK
 
 
-class FixedSizedSedes(BaseSedes[TSerializable, TDeserialized]):
+class BasicSedes(BaseSedes[TSerializable, TDeserialized]):
 
-    def __init__(self, length: int):
-        if length <= 0:
+    def __init__(self, size: int):
+        if size <= 0:
             raise ValueError("Length must be greater than 0")
 
-        self.length = length
+        self.size = size
+
+    #
+    # Size
+    #
+    is_static_sized = True
```

Comment:
Not sure if it's a good pattern to overwrite `@property def BaseSedes.is_static_sized()` like this.
Alternatively, maybe add `_is_static_sized` and let `is_static_sized` return `_is_static_sized`?

Reply:
Convinced, changed it to a property.